lib/dialogs/settings-group.tsx

```diff
@@ -47,7 +48,8 @@ export const SettingsGroup = ({
                 groupSlug,
                 slug,
                 title,
-                isEnabled: slug === activeSlug,
+                isEnabled:
+                  isEnabled === undefined ? slug === activeSlug : isEnabled,
```

Comment:
we can use the `??` operator to pick an alternative value if `isEnabled` is `null` or `undefined`

```ts
isEnabled: isEnabled ?? slug === activeSlug,
```